src/cmd/top/joblist_pane.c

```diff
@@ -29,6 +29,9 @@ struct joblist_pane {
     json_t *jobs;
     struct ucache *ucache;
 
+    const char *queue;
+    bool show_queue;
+
```

Comment:
you have this in the ` flux-mini: remove -q short option for --quiet ` commit, I guess a merge conflict.